### PR TITLE
feat(wallet_ffi): add get utxos method

### PR DESCRIPTION
**Pull request by stringhandler**
It adds a method

### CODE REVIEW

File: applications/tari_console_wallet/src/automation/commands.rs [hunk 1]
```diff
@@ -711,7 +711,7 @@ pub async fn command_runner(
                 println!("Total value of UTXOs: {}", sum);
             },
             CountUtxos => {
-                let utxos = output_service.get_unspent_outputs().await?;
+                let utxos = output_service.get_unspent_outputs(0,1000,false).await?;
```
Comment (SWvheerden) [inline applications/tari_console_wallet/src/automation/commands.rs:714]: Is there a reason this is left filled in, and not taken from user commands?

File: applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs [hunk 2]
```diff
@@ -221,7 +221,7 @@ impl wallet_server::Wallet for WalletGrpcServer {
     ) -> Result<Response<GetUnspentAmountsResponse>, Status> {
         let mut output_service = self.get_output_manager_service();
         let unspent_amounts;
-        match output_service.get_unspent_outputs().await {
+        match output_service.get_unspent_outputs(0, 1000, false).await {
```
Comment (SWvheerden) [inline applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs:224]: same here

**Review by SWvheerden — COMMENTED** [review]
This looks good. 
The only question I have is about the fixed values for the commands and GRPC requests?

File: applications/tari_console_wallet/src/automation/commands.rs (hunk 1 above)
Comment (stringhandler) [reply applications/tari_console_wallet/src/automation/commands.rs:714]: I update this after #4612 is merged

**Pull request closed without merging**